### PR TITLE
fix: check if assignment of power factor is correct

### LINKED ISSUE #113: fix: check if assignment of power factor is correct
In PF:
* inductive = 0
* capacitive = 1

but in dependency of typ (consumer or producer) it is overexcited or underexcited.

Need to check!

### PR DESCRIPTION
**Pull request by SebastianDD**
Fixes #113

### CODE REVIEW

File: powerfactory_tools/exporter/exporter.py [hunk 1]
```diff
@@ -2263,7 +2261,7 @@ def calc_load_mv_power_asym(self, load: PFTypes.LoadMV) -> LoadMV:
         # in PF for consumer: ind. cosphi = under excited; cap. cosphi = over excited
         cosphi_dir_cons = CosphiDir.UE if load.pf_recap == 0 else CosphiDir.OE
         # in PF for producer: ind. cosphi = over excited; cap. cosphi = under excited
-        cosphi_dir_prod = CosphiDir.OE if load.pfg_recap == 0 else CosphiDir.UE
+        cosphi_dir_prod = CosphiDir.UE if load.pfg_recap == 1 else CosphiDir.OE
```
Comment (sasanjac) [inline powerfactory_tools/exporter/exporter.py:2264]: isn't this the same logic as before?

Comment (sasanjac) [reply powerfactory_tools/exporter/exporter.py:2264]:
```suggestion
        cosphi_dir_prod = CosphiDir.UE if load.pfg_recap else CosphiDir.OE
```

File: powerfactory_tools/exporter/exporter.py [hunk 2]
```diff
@@ -2412,8 +2410,8 @@ def create_q_controller_builtin(  # noqa: PLR0911
 
         if av_mode == LocalQCtrlMode.COSPHI_CONST:
             cosphi = gen.cosgini
-            cosphi_dir = CosphiDir.UE if gen.pf_recap == 1 else CosphiDir.UE
-            q_controller: ControlType = ControlCosphiConst(
+            cosphi_dir = CosphiDir.UE if gen.pf_recap == 1 else CosphiDir.OE
```
Comment (sasanjac) [inline powerfactory_tools/exporter/exporter.py:2413]:
```suggestion
            cosphi_dir = CosphiDir.UE if gen.pf_recap else CosphiDir.OE
```

File: powerfactory_tools/exporter/exporter.py (hunk 1 above)
Comment (SebastianDD) [reply powerfactory_tools/exporter/exporter.py:2264]: yes, this was not a fix , just a design change as check/assigment is at all places performed like this

**Pull request merged**